overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -52226,7 +52226,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Imagine all the people",
-          "desc:8": "Peace, bro.\n\nLook at the Rainbows...\n\n\n\nDaaaamn!"
+          "desc:8": "Peace, bro.\n\nLook at the Rainbows...\n\n\n\nDaaaamn!\nEventually you will need a total of 12 Peace Essences. Might want to use the Industrial Forgoing Mob Duplicator to help you out collecting syringe samples."
```

Comment:
why specify IF Mob Duplicator? EnderIO or RFTools works about as well, and Roots can summon most of the mobs.
also, typo in Foregoing

Reply:
I specified the mob duplicator because its the best option to spawn extra mobs of any type you want and it has no mention anywhere else. I could mention all the options here, but I dont feel thats really needed.

Reply:
dont mention any, let the player choose

Reply:
Sure, can do.